policyengine_uk/variables/contrib/labour/private_school_vat.py

```diff
@@ -0,0 +1,40 @@
+from policyengine_uk.model_api import *
+from sys import stderr
+
+
+class private_school_vat(Variable):
+    label = "Private school VAT"
```

Comment:
`label` should start lower-case (you should be able to drop `label` mid-sentence)